### PR TITLE
Assertions mod

### PR DESCRIPTION
**Pull request by fluidvanadium**
these are assertions about whether a proposal was fulfilled, packaged up nicely

### CODE REVIEW

File: zingolib/src/wallet/notes/query.rs [hunk 1]
```diff
@@ -38,13 +38,13 @@ impl OutputSpendStatusQuery {
 pub struct OutputPoolQuery {
     /// will the query include transparent notes? (coins)
     #[getset(get = "pub")]
-    transparent: bool,
+    pub transparent: bool,
```
Comment (Oscar-Pepper) [inline zingolib/src/wallet/notes/query.rs:41]: i dont understand why the pub getset isnt good enough and how making these pub relates to this PR? i probably have missed something

**Review by Oscar-Pepper — DISMISSED** [review]

File: zingolib/src/wallet/notes/query.rs (hunk 1 above)
Comment (fluidvanadium) [reply zingolib/src/wallet/notes/query.rs:41]: its so that we can use naive struct instantiation

Comment (fluidvanadium) [reply zingolib/src/wallet/notes/query.rs:41]: oh, i guess i didnt end up using it in this branch ...

Comment (AloeareV) [reply zingolib/src/wallet/notes/query.rs:41]: Initializing an `OutputPoolQuery` with struct literal notation, i.e. 
```rust
let opq = OutputPoolQuery {
    transparent: true,
    sapling: true,
    orchard: false,
}
```
is clearer to me than something like `OutputPoolQuery::new(true, true, false)` or `OutputPoolQuery::default().set_transparent(true)...`

**Review by zancas — APPROVED** [review]

**Review by zancas — APPROVED** [review]

**Pull request merged**